src/main/java/teammates/common/datatransfer/SqlCourseRoster.java

```diff
@@ -126,8 +128,8 @@ public static Map<String, List<Student>> buildTeamToMembersTable(List<Student> s
      */
     public ParticipantInfo getInfoForIdentifier(String identifier) {
         String name = Const.USER_NOBODY_TEXT;
-        String teamName = Const.USER_NOBODY_TEXT;
-        String sectionName = Const.DEFAULT_SECTION;
+        Team team = new Team(null, Const.USER_NOBODY_TEXT);
+        Section section = Const.DEFAULT_SECTION_ENTITY;
```

Comment:
same, should instantiate Section here instead of in Const. use the old `Const.DEFAULT_SECTION`. rationale explained below